apps/src/p5lab/P5Lab.js

```diff
@@ -474,6 +474,31 @@ P5Lab.prototype.init = function(config) {
         ];
       }
     });
+  } else if (this.isSpritelab) {
+    // check if initialAnimationList has backgrounds. If it doesn't add it from defaultSprites.json
+    let hasBackgrounds = false;
+    let configDictionary = {};
```

Comment:
this same logic to create `configDictionary` is in the conditional above, so we should move it into a helper function